x-pack/plugins/ml/public/application/timeseriesexplorer/components/plot_function_controls/plot_function_controls.tsx

```diff
@@ -30,11 +34,57 @@ const plotByFunctionOptions = [
 export const PlotByFunctionControls = ({
   functionDescription,
   setFunctionDescription,
+  selectedDetectorIndex,
+  selectedJobId,
+  selectedEntities,
 }: {
   functionDescription: undefined | string;
   setFunctionDescription: (func: string) => void;
+  selectedDetectorIndex: any;
+  selectedJobId: string;
+  selectedEntities: Record<string, any>;
```

Comment:
Could it be a more specific type?

Reply:
Updated here https://github.com/elastic/kibana/pull/83507/commits/607d7880884564514d8fb34b4972bc4ab644ea75

Reply:
@qn895 can't find it in the commit hash you've mentioned

Reply:
Not sure why Github is being weird but it's `https://github.com/elastic/kibana/pull/83507/commits/607d7880884564514d8fb34b4972bc4ab644ea75`